_pages/oidc.md

```diff
@@ -36,17 +36,16 @@ sidenav:
 
 ## Getting started
 
-### Choosing an authentication method
+### Client authentication
 
-Login.gov supports two ways of authenticating clients: **private_key_jwt** and **PKCE**.
+Login.gov supports **private_key_jwt** client authentication or no authentication.
 
 - **private_key_jwt** (preferred for **web apps**)
   The client sends a [JSON Web Token][jwt], or JWT, signed with a private key (minimum length of 2048 bits) when requesting access tokens. The corresponding public key is registered with the IdP ahead of time, similar to SAML.
+- **none** (for **native mobile apps**)
```

Comment:
I see how this is technically correct, but I think in terms of implementation, it may be misleading and confusing based on the rest of the page, and our expected workflows

Since the rest of the page has toggles for private_key_jwt/PKCE (see example below) I think this might be preferable?

```suggestion
- *none* with **PKCE** (for **native mobile apps**)
```

| example |
| --- |
|  <img width="582" alt="Screen Shot 2022-02-23 at 11 58 11 AM" src="https://user-images.githubusercontent.com/458784/155398363-cfb05d8d-eb3f-433a-ac24-eef3c407c780.png"> |

Reply:
alright, I can live with that. I'm going to get in touch on HackerOne about something related though. Do you see those reports?

Reply:
Yes, we do see the HackerOne reports

Reply:
I just tried and it says the program is disabled. I can just email instead